packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -22,6 +22,7 @@ import {
   ActiveNavElement,
   NavElement,
 } from '../types';
+import { Atlas, RealmActive, RealmInactive, Charts } from '../helpers/Icons';
```

Comment:
Wonder if we should suffix all these names with `Icon` to clarify their usage.

Reply:
👍